sp/src/materialsystem/stdshaders/SDK_eyes_vs20.fxc

```diff
@@ -0,0 +1 @@
+SDK_Eyes_vs20.fxc
```

Comment:
With this link in place it will create two shaders named `SDK_Eyes_vs20` and `SDK_eyes_vs20` respectively. I don't know which of these is the canonical/proper name so building both should work in either case.
I'm not sure if this works out on Windows though. On Linux this file is "missing" because of case sensitivity. I don't know if shader names are case sensitive. Renaming and fixing up the references will also work, but I'll need to know which one is the correct name.

Reply:
The "official" (non-SDK_) shader is lower-case "eyes". I'm guessing that's the right one and will just rename the file.

Reply:
I checked HL2's `stdshader_dx9.so`, where it's listed as `eyes_vs20`. I think that's a pretty good indicator it should be lower-case, if it makes any difference at all.